tests/test_component_compile.py

```diff
@@ -14,15 +18,14 @@ def _prepare_component(tmp_path, component_name='test-component'):
     test_run_component_new_command(tmp_path=tmp_path)
 
     with open(tmp_path / 'dependencies' / component_name / 'component/main.jsonnet', 'a') as file:
-        file.write('''
-{
-  "test_service_account": kube.ServiceAccount('test') {
-    metadata+: {
-      namespace: params.namespace,
-    },
-  },
-}
-''')
+        file.write(dedent('''
+            {
```

Comment:
Indentation broke a little here.

Reply:
Yep, same problem as above. I'll fix it.

Good catch!

Reply:
Done.